src/hex/mod.rs

```diff
@@ -59,6 +59,10 @@ use std::{
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "packed", repr(C))]
 #[cfg_attr(feature = "bevy_reflect", derive(bevy_reflect::Reflect))]
+#[cfg_attr(
+    feature = "bevy_ecs",
+    derive(bevy_ecs::resource::Resource, bevy_ecs::component::Component)
```

Comment:
```suggestion
    derive(bevy_ecs::component::Component)
```
I don't see any use for a single hexagonal coordinate to be a bevy resource.
I'm not very fond of adding the component derive either but it seems to be often requested so for this type I guess it makes sense